AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -17,6 +17,8 @@ define([
             tabImagesLimit: 4,
             serieFilterValue: '',
             modelFilterValue: '',
+            isSeriesTabVisible: [],
+            isModelTabVisible: [],
```

Comment:
Do you think this intermidiate fields are necessary? Why wouldn't `showSeriesTab` method work directly with `this.relatedImages().series[record.id].length !== 0 || typeof this.relatedImages().series[record.id] === 'undefined'`